src/components/Flyout.tsx

```diff
@@ -0,0 +1,43 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { AppDispatch, RootState } from '../redux/store';
+import { clearSelection } from '../redux/slices/selectedItemsSlice';
+import csvCreate from '../scripts/csvCreate';
+
+function Flyout() {
+  const dispatch = useDispatch<AppDispatch>();
+
+  const selectedItems = useSelector(
+    (state: RootState) => state.selectedItems.selectedGifs
+  );
+  const selectedNumber = selectedItems.length;
+  function unselect() {
+    dispatch(clearSelection());
+  }
+
+  const downloadCSV = () => {
+    const csvContent = csvCreate(selectedItems);
+    const blob = new Blob([csvContent], { type: 'text/csv;charset=utf-8;' });
+    const url = URL.createObjectURL(blob);
+    const link = document.createElement('a');
+    link.setAttribute('href', url);
+    link.setAttribute('download', `${selectedItems.length}_gifs.csv`);
+    document.body.appendChild(link);
+    link.click();
+    document.body.removeChild(link);
+    URL.revokeObjectURL(url);
+  };
+
+  return selectedNumber > 0 ? (
```

Comment:
just for easier reading:
```suggestion
  If (! selectedNumber) return null;
  return (<div className="flyout">...</div>);
```